packages/playground/src/weblets/tf_contracts_list.vue

```diff
@@ -81,10 +81,10 @@
       </v-btn>
     </template>
   </weblet-layout>
-
   <v-dialog width="70%" v-model="deletingDialog">
     <v-card>
-      <v-card-title class="text-h5"> Are you sure you want to delete the following contracts? </v-card-title>
+      <v-card-title class="text-h5 mt-2"> Are you sure you want to delete the following contracts? </v-card-title>
```

Comment:
Please confirm if you wish to proceed with the deletion of the following contracts.

Reply:
I think we need to open an issue for this one, the change I made here is just to add some space before it.